README.md

```diff
@@ -1,4 +1,4 @@
-This README is just a fast *quick start* document. You can find more detailed documentation at http://redis.io.
+This README is just a fast *quick start* document. You can find more a more detailed documentation [here][3].
```

Comment:
Please don't link `here`. Keep the full URL.
also the sentence is wrong (double `more` now)